src/tests/tablet/schedule/AUserCannotUpdateIfDoesnotClickUpdate.java

```diff
@@ -0,0 +1,86 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import framework.common.MeetingMethods;
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.rest.RootRestMethods;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC45: Verify if the user doesn't click update button the information is not updated
+ * @author Asael Calizaya
+ *
+ */
+public class AUserCannotUpdateIfDoesnotClickUpdate {
```

Comment:
The title is a little ambiguous is should be ....AMeetingIsNotUpdatedWhenUpdatedButtonIsNotClicked